README.md

```diff
@@ -62,17 +65,19 @@ app
 The client side code to support this is very simple too:
 
 ```html
-<script src="node_modules/koa-web-monetization/client.js"></script>
+<body>
+  <h1>Hello World</h1>
+  <p>Underneath this is a paid image. It will load if you have web monetization on.</p>
+  <div id="container">
+    <!-- Paid image will be appended here -->
+  <img src="/content" width="600"/>
+  </div>
+
+</body>
+<script src="/client.js"></script>
```

Comment:
It's a big strange to have tags outside of the body; can't these go in a `<head>`?